src/en/credentials.md

```diff
@@ -417,18 +425,31 @@ Credentials:
 
 ### Removing local credentials
 
-If a local credential (i.e. not cached on a controller) is no longer required,
-it can be removed:
+The `remove-credential` command is used to remove a local credential (i.e. not
+cached on a controller):
 
 ```bash
 juju remove-credential aws bob
 ```
 
+### Setting a remote credential for a model
+
+To set what remote credential should be used for a model the `set-credential`
```

Comment:
The command is named 'set' but the functionality that it does is really more along the lines of 'change' or 'replace'.

Reply:
Will reword. BTW, isn't the functionality of `update-credential` contained within this new command?

Reply:
No, 'set-credential' will upload a credential from local store (client) if it does not exist remotely on the controller. It will not update the content of remotely stored credential, only 'update-credential' command can do that.